packages/common/src/services/audius-backend/solana.ts

```diff
@@ -430,6 +431,92 @@ export const createTransferToUserBankTransaction = async (
   return tx
 }
 
+export const createFundAccountAndPurchaseTransaction = async (
```

Comment:
this won't work, right? we can't transfer the funds to the userbank in the coinflow flow?

Reply:
is it ok if I pull this out? Thinking best to merge this with the working stuff. Can revert my commit later to do the add funds flow